backend/routes.py

```
@@ -35,6 +48,195 @@ def register_event_handlers():
     event_bus.register_handler(EventType.PR_COMMENTED, sample_handler, PlatformType.GITHUB)
     event_bus.register_handler(EventType.PR_MERGED, sample_handler, PlatformType.GITHUB)
 
+@router.post("/repo-stats")
+@limiter.limit("10/minute")
+async def get_repo_stats(
+    request: RepoRequest,
+    current_user: UUID = Depends(get_current_user)
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)